### ISSUE TITLE
[webview_flutter] The click event is not pressed on iOS 18.2.

### ISSUE THREAD

**Issue by ttb-inc** [flutter/flutter#160900]
### Steps to reproduce

See the sample code.

1. Try pressing the button in the webview (checks OK)
2. Try pressing the FloatingActionButton '+' (UI event)
3. Try pressing the button in the webview again (to check for errors).
4. After pressing the 'Reload' button, reload the webview.
5. Try pressing the button in the webview (checks OK)

### Expected results

...

### Actual results

After the event of the flutter Button UI is pressed, the event of the webview is not pressed.

The error occurs only on iOS 18.2.
Works fine on iOS 18.1.1 and earlier.

If I'm doing something wrong or need additional options, please help.
Thanks a lot.

### Code sample

<details open><summary>Code sample</summary>

```dart
import 'package:flutter/material.dart';
import 'package:flutter_inappwebview/flutter_inappwebview.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
        useMaterial3: true,
      ),
      home: const MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({super.key, required this.title});

  final String title;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  late InAppWebViewController _webviewController;

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
        title: Text(widget.title),
        leading: FloatingActionButton(
          onPressed: () {
            ///
            /// After reloading, the webview's events(e.g. onclick) will work again.
            ///
            _webviewController.reload();
          },
          child: const Icon(Icons.refresh),
        ),
      ),
      body: InAppWebView(
        initialUrlRequest: URLRequest(
          url: WebUri('https://google.com/'),
        ),
        onWebViewCreated: (controller) {
          _webviewController = controller;
        },
      ),
      floatingActionButton: FloatingActionButton(
        onPressed: () {
          ///
          /// When a button input event is pressed, the webview's events(e.g. onclick) will stop working.
          ///
        },
        child: const Icon(Icons.add),
      ), // This trailing comma makes auto-formatting nicer for build methods.
    );
  }
}
```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
[!] Flutter (Channel [user-branch], 3.24.5, on macOS 15.1.1 24B91 darwin-arm64, locale ko-KR)
    ! Flutter version 3.24.5 on channel [user-branch] at /Users/jeonglockpark/Desktop/work/flutter_sdk/flutter
      Currently on an unknown channel. Run `flutter channel` to switch to an official channel.
      If that doesn't fix the issue, reinstall Flutter by following instructions at https://flutter.dev/setup.
    ! Upstream repository unknown source is not a standard remote.
      Set environment variable "FLUTTER_GIT_URL" to unknown source to dismiss this error.
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 16.2)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2024.2)
[✓] VS Code (version 1.96.2)
```

</details>

**Comment by ttb-inc** [comment]
I'm using several flutter UIs on webview.
I'm experiencing a lot of issues.
I don't know if this is an iOS 18.2 issue, flutter SDK issue, or webview_flutter package issue.
Thank you.

**Comment by Manuito83** [comment]
@ttb-inc this was corrected in 3.27.1. Be aware of graphic issues with Impeller, though.
